src/l10n/fr.js

```diff
@@ -0,0 +1,357 @@
+(function() {
+    let translations = {
+            'Passwords'                           : 'Mots de passe',
+            'All'                                 : 'Tous',
+            'Folders'                             : 'Dossiers',
+            'Recent'                              : 'Récents',
+            'Favorites'                           : 'Favoris',
+            'Shared'                              : 'Partagés',
+            'Tags'                                : 'Mots clés',
+            'Security'                            : 'Sécurité',
+            'Trash'                               : 'Corbeille',
+            'Details'                             : 'Détails',
+            'Edit'                                : 'Modifier',
+            'Rename'                              : 'Renommer',
+            'Delete'                              : 'Effacer',
+            'Restore'                             : 'Restaurer',
+            'New Password'                        : 'Nouveau mots de passe',
```

Comment:
This entry is to create a new password, so it should be singular => `Nouveau mot de passe`